include/mgos_ads1x1x.h

```diff
@@ -119,6 +157,11 @@ bool mgos_ads1x1x_read(struct mgos_ads1x1x *dev, uint8_t chan, int16_t *result);
  */
 bool mgos_ads1x1x_read_diff(struct mgos_ads1x1x *dev, uint8_t chanP, uint8_t chanN, int16_t *result);
 
+/* Centralised helper function to write configuration to an ADS1119
+* Sends the command for WREG first
+*/
+bool mgos_ads1119_write_conf(struct mgos_ads1x1x * dev, uint8_t value);
+
```

Comment:
helpers do not go into the public API, as they are not meant to be called by users. This should go into a src/*_internal.h file.